04-single-hop-6lowpan-icmp/test_spec04.py

```diff
@@ -113,6 +113,11 @@ def test_task04(riot_ctrl):
                 count=10000, interval=100, packet_size=100)
     assert res['stats']['packet_loss'] < 10
 
+    # enforce reconnect to pinged's terminal as connection to it in the IoT-LAB
+    # sometimes get's lost silently in the CI after the 15 min of pinging
+    pinged.stop_term()
```

Comment:
From offline discussion with @fjmolinas: we might get this working by moving this above l112, as the connection from the IoT-LAB side might still be perceived as open after this and this the `start_term()` fails silently. By moving this above the ping there is enough time for the connection to time out properly.

Reply:
I started [a run](https://github.com/miri64/RIOT/runs/951201329) containing [this PR](https://github.com/miri64/RIOT/runs/951201329?check_suite_focus=true#step:3:184). Let's see if this theory turns out true